src/AppBundle/Resources/assets/scss/digideps/_cards.scss

```diff
@@ -2,23 +2,23 @@
 // CARDS
 // The report card styling on the 'Your reports' page
 
-.card {
+.opg-card {
     position: relative;
     box-sizing: border-box;
     min-height: 120px;
     padding: 15px;
```

Comment:
Should use `govuk-spacing(3)` rather than hard-coding a pixel value (so spacing is consistent throughout the app)